nixos/modules/config/sysctl.nix

```diff
@@ -72,5 +72,9 @@ in
     # Disable YAMA by default to allow easy debugging.
     boot.kernel.sysctl."kernel.yama.ptrace_scope" = mkDefault 0;
 
+    # Improve compatibility with applications that allocate
+    # a lot of memory, like modern games
+    boot.kernel.sysctl."vm.max_map_count" = mkDefault 1048576;
+
```

Comment:
extra line

Reply:
It was already there. (Notice that there isn't a new line *before*)
Unless it was already wrong before...

Reply:
Feel free to drop it if you want or not